File: mobile/src/main/java/com/zacharytamas/often/models/HabitOccurrence.java
package com.zacharytamas.often.models;

import com.orm.SugarRecord;

import java.util.Date;

/**
 * Created by zacharytamas on 10/25/15.
 */
public class HabitOccurrence extends SugarRecord<HabitOccurrence> {
    public Habit habit;
    public int streakLength;
    public Date completedAt;
    public Date wasDueAt;
}

File: mobile/src/main/java/com/zacharytamas/often/models/HabitOccurrence.kt
package com.zacharytamas.often.models

import com.orm.SugarRecord
import java.util.*

/**
 * Created by zacharytamas on 10/9/15.
 */
public class HabitOccurrence : SugarRecord<HabitOccurrence>() {
    
    var habit: Habit? = null
    var streakLength: Int = 0
    var completedAt: Date? = null
    var wasDueAt: Date? = null

}